web/src/components/molecules/Asset/AssetListTable/index.tsx

```diff
@@ -82,38 +80,36 @@ const AssetListTable: React.FC<AssetListTableProps> = ({
           <Icon icon="edit" />
         </Link>
       ),
+      align: "center",
     },
     {
       title: () => <Icon icon="message" />,
       dataIndex: "commentsCount",
       key: "commentsCount",
       render: (_, asset) => {
         return (
-          <Button type="link" onClick={() => onAssetSelect(asset.id)}>
+          <CommentsButton type="link" onClick={() => onAssetSelect(asset.id)}>
             <CustomTag
               value={asset.comments?.length || 0}
               color={asset.id === selectedAsset?.id ? "#87e8de" : undefined}
             />
-          </Button>
+          </CommentsButton>
         );
       },
+      align: "center",
     },
     {
       title: t("File"),
       dataIndex: "fileName",
       key: "NAME",
       sorter: true,
-      defaultSortOrder:
-        sort?.type === "NAME" ? (sort.direction === "ASC" ? "ascend" : "descend") : null,
```

Comment:
Why did you remove the default sort order?

Reply:
I didn't think it needed to be displayed as well as the Content table, should I change it back?

Reply:
Okay if it is not displayed then it is okay to remove it.